app/serializers/rest/admin/account_serializer.rb

```diff
@@ -83,4 +83,8 @@ def ips
   def ip
     ips&.first&.ip
   end
+
+  def media_attachments
+    object.media_attachments&.sum(:file_file_size)
```

Comment:
[Account#media_attachments](https://github.com/casaper/mastodon/blob/14f436c457560862fafabd753eb314c8b8a8e674/app/models/concerns/account_associations.rb#L31) is a relation. Safe navigation is thus redundant because it cannot be nil.

```suggestion
    object.media_attachments.sum(:file_file_size)
```

Reply:
thanks, i wasn't sure. it did work without it,